### PR TITLE
Schema and event triggers page update

### PR DESCRIPTION
**Pull request by surendran82**
fix int-#76
### Description
Updated landing for remote schema and event trigger 

What component does this PR affect? 

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by tirumaraiselvan** [comment]
@surendran82 can you close this DRAFT PR and open a normal one?

**Comment by coco98** [comment]
Suggestions:
1. Remove first section on what is remote schema if there's >=1 remote schema. Same for event triggers
2. The glitch page leads to the readme, which is empty. Let's put some instructions there.
screenshot 2019-03-05 at 3 37 03 pm

File: console/src/components/Services/CustomResolver/Landing/CustomResolver.js [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+    // const lightGrayArrow = require('./light-gray-arrow.svg');
+    // const darkGrayArrow = require('./dark-gray-arrow.svg');
+
     // const landingImage = require('./schema-stitching-color.png');
     // const landingImage = 'https://storage.googleapis.com/hasura-graphql-engine/console/assets/schema-stitching-diagram.png';
 
-    const { dispatch, migrationMode } = this.props;
+    const { dispatch, migrationMode, customResolverList } = this.props;
+    const showFirstSection = customResolverList.resolvers.length ? false : true;
+    console.log(showFirstSection);
```
Comment (praveenweb) [inline console/src/components/Services/CustomResolver/Landing/CustomResolver.js]: Remove console.log

File: console/src/components/Services/CustomResolver/Landing/CustomResolver.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
-
+import TopicDescription from '../../CommonLanding/TopicDescription';
+import TryItOut from '../../CommonLanding/TryItOut';
 class CustomResolver extends React.Component {
   render() {
     const styles = require('../Styles.scss');
+    const node = require('./Node.svg');
+    const Rectangle = require('./Rectangle.svg');
+
+    // const lightGrayArrow = require('./light-gray-arrow.svg');
```
Comment (praveenweb) [inline console/src/components/Services/CustomResolver/Landing/CustomResolver.js]: remove source image if its part of code

**Pull request merged**